internal/reconciliations/gateway/dns_entry.go

```diff
@@ -77,10 +78,9 @@ func deleteDnsEntry(ctx context.Context, k8sClient client.Client, name, namespac
 }
 
 func fetchIstioIngressGatewayIp(ctx context.Context, k8sClient client.Client) (string, error) {
-
 	istioIngressGatewayNamespaceName := types.NamespacedName{
-		Name:      "istio-ingressgateway",
-		Namespace: "istio-system",
+		Name:      KymaGatewayName,
```

Comment:
This needs to be reverted since it is not correct and will fail the reconciliation on Gardener clusters.

Reply:
This is kind of a mistake I guess.